### ISSUE TITLE
A place to chat/discuss about VS Code.

### ISSUE THREAD

**Issue by trusktr** [microsoft/vscode#113561]
I can't seem to find one.

Maybe it would be worth opening up the Discussions tab here on GitHub?

**Comment by nrayburn-tech** [comment]
In the readme, this is linked https://gitter.im/Microsoft/vscode.  I have never used it, so I don't really know how active/useful it is though.

**Comment by djmisterjon** [comment]
yes! i have nothing vs Stackoverflow but centralizing issues, supports and discutions about VsCode at the same place would be ideal I think.
maybe need poke the Admin of this repo !

**Comment by NSExceptional** [comment]
+1 for enabling Discussions on this and all related repos. Would make SEO for the rest of us a lot easier than searching StackOverflow and Reddit and whatever else.

**Comment by djmisterjon** [comment]
The irony is that Github is belongs to Microsoft now and it does not use its own tools !
If i choose Microsoft products since many years it to ideally centralize as much as possible my all tasks and link all to one entities.

**Comment by devmahdi404** [comment]
any plan on enabling **Github discussions**?

**Comment by devmahdi404** [comment]
https://github.com/microsoft/vscode-discussions

**Comment by djmisterjon** [comment]
the issue is github discutions are not indexed by google.
So you wast time to answer people.

**Comment by digitarald** [comment]
Since [vscode-discussions](https://github.com/microsoft/vscode-discussions) got linked, I wanted to give some background: We launched it at small scale to experiment, solely focused on extension development (for now). If that is what you want to talk about, learn and collaborate with others, please swing by.

So far we have seen, Discussions have great SEO (afaik not as Discussions launched, but a lot changed since then).

When we'll grow the topic surface beyond extension development, I'll update this thread.
